reference/7/Microsoft.PowerShell.Utility/Get-Error.md

```diff
@@ -0,0 +1,144 @@
+    StackTrace           :
+   at System.Management.Automation.SessionStateInternal.GetChildItems(String path, Boolean recurse, UInt32 depth,
+CmdletProviderContext context)
+   at System.Management.Automation.ChildItemCmdletProviderIntrinsics.Get(String path, Boolean recurse, UInt32
+depth, CmdletProviderContext context)
+   at Microsoft.PowerShell.Commands.GetChildItemCommand.ProcessRecord()
+    Message              : Cannot find path 'C:\NoRealDirectory' because it does not exist.
+    Source               : System.Management.Automation
+    HResult              : -2146233087
+TargetObject          : C:\NoRealDirectory
+CategoryInfo          : ObjectNotFound: (C:\NoRealDirectory:String) [Get-ChildItem], ItemNotFoundException
+FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
+InvocationInfo        :
+    MyCommand        : Get-ChildItem
+    ScriptLineNumber : 1
+    OffsetInLine     : 1
+    HistoryId        : 57
+    Line             : Get-Childitem -path c:\NoRealDirectory
+    PositionMessage  : At line:1 char:1
+                       + Get-Childitem -path c:\NoRealDirectory
+                       + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    InvocationName   : Get-Childitem
+    CommandOrigin    : Internal
+ScriptStackTrace      : at <ScriptBlock>, <No file>: line 1
+PipelineIterationInfo :
+```
+
+### Example 2: Get the specified number of error messages that occurred in the current session
+
+This example shows how to use `Get-Error` with the **Newest** parameter. In this example, **Newest**
+returns the details of the 3 newest errors that occurred in this session.
+
+```powershell
+Get-Error -Newest 3
+```
+
+### Example 3: Send a collection of errors to receive detailed messages
+
+The **$Error** automatic variable contains an array of error objects in the current session. The
+array of objects can be piped to `Get-Error` to receive detailed error messages.
+
+In this example, **$Error** is piped to the 'Get-Error' cmdlet. the result is list of detailed error
+messages, similar to the result of Example 1.
+
+```powershell
+$Error | Get-Error
+```
+
+## PARAMETERS
+
+### -Newest
+
+Specifies the number of errors to display that have occurred in the current session.
+
+```yaml
+Type: PSExtendedError
+Parameter Sets: (All)
+Aliases: None
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### Pipeline input
+
+`Get-Error` accepts pipeline input. For example, `$Error | Get-Error`.
+
+## OUTPUTS
```

Comment:
Output is a PSExtendedError object

Reply:
This is not formatted correctly per the PlatyPS schema. See https://github.com/PowerShell/platyPS/blob/master/platyPS.schema.md